src/vcpkg/build.cpp

```diff
@@ -979,7 +979,19 @@ namespace vcpkg
                 error_logs = fs.read_lines(logs).value_or_exit(VCPKG_LINE_INFO);
                 Util::erase_remove_if(error_logs, [](const auto& line) { return line.empty(); });
             }
-            return ExtendedBuildResult{BuildResult::BUILD_FAILED, stdoutlog, std::move(error_logs)};
+            ExtendedBuildResult result{BuildResult::BUILD_FAILED, stdoutlog, std::move(error_logs)};
+            const auto user_required_path =
+                buildpath / Strings::concat("required-user-interaction-", action.spec.triplet(), ".txt");
+            if (fs.exists(user_required_path, VCPKG_LINE_INFO))
+            {
+                result.user_required_interaction = fs.read_contents(user_required_path, VCPKG_LINE_INFO);
+            }
+            const auto user_hints_path = buildpath / Strings::concat("user-hints-", action.spec.triplet(), ".txt");
+            if (fs.exists(user_hints_path, VCPKG_LINE_INFO))
```

Comment:
Better make sure that it's a regular file

Reply:
Since it is only an internal interface I think it is ok here. Crashing in case of an folder is a good thing. And I don't care if someone wants to use symbolic links in the future.